html/pfappserver/lib/pfappserver/Form/Config/Pf.pm

```diff
@@ -235,11 +241,50 @@ sub field_list {
                     });
             }
         }
+        if (my $validate_method = $self->validator_for_field($doc_section_name)) {
+            $field->{validate_method} = $validate_method;
+        }
         push(@$list, $name => $field);
     }
     return $list;
 }
 
+=head2 validator_for_field
+
+Get the validator for a field
+
+=cut
+
+sub validator_for_field {
+    my ($self, $field) = @_;
+
+    return {
+        "fencing.passthroughs" => sub { validate_fqdn_not_in_passthroughs(@_, "passthroughs") },
+        "fencing.isolation_passthroughs" => sub { validate_fqdn_not_in_passthroughs(@_, "isolation_passthroughs") },
```

Comment:
This would be better as static lookup hash to avoid creating the same data multiple times.

Reply:
Also there should also be a check if someone changes the hostname or domain to a current passthrough domain.